updating/updating-cluster-within-minor.adoc

```diff
@@ -1,11 +1,11 @@
-[id="updating-cluster-between-minor"]
-= Updating a cluster between minor versions
+[id="updating-cluster-within-minor"]
+= Updating a cluster within a minor version using the web console
 include::modules/common-attributes.adoc[]
-:context: updating-cluster-between-minor
+:context: updating-cluster-within-minor
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster between minor versions.
+You can update, or upgrade, an {product-title} cluster by using the web console. The following steps are updating a cluster within a minor a version. You can use the same instructions for updating a cluster between minor versions.
```

Comment:
@LalatenduMohanty Could you take a look at this and make sure this is accurate.

Reply:
Right. The steps to use web console for upgrading a cluster is same for z stream as well as y stream upgrades.